src/runtime/c/lib/util/message.c

```diff
@@ -1,84 +1,74 @@
-/* 
- * Copyright (c) 2017-2018, SyLabs, Inc. All rights reserved.
- * Copyright (c) 2017, SingularityWare, LLC. All rights reserved.
+/*
+ * Copyright (c) 2018, Sylabs, Inc. All rights reserved.
  *
- * Copyright (c) 2015-2017, Gregory M. Kurtzer. All rights reserved.
- * 
- * Copyright (c) 2016-2017, The Regents of the University of California,
- * through Lawrence Berkeley National Laboratory (subject to receipt of any
- * required approvals from the U.S. Dept. of Energy).  All rights reserved.
- * 
- * This software is licensed under a customized 3-clause BSD license.  Please
- * consult LICENSE.md file distributed with the sources of this project regarding
- * your rights to use or distribute this software.
- * 
- * NOTICE.  This Software was developed under funding from the U.S. Department of
- * Energy and the U.S. Government consequently retains certain rights. As such,
- * the U.S. Government has been granted for itself and others acting on its
- * behalf a paid-up, nonexclusive, irrevocable, worldwide license in the Software
- * to reproduce, distribute copies to the public, prepare derivative works, and
- * perform publicly and display publicly, and to permit other to do so. 
- * 
+ * This software is licensed under a 3-clause BSD license. Please
+ * consult LICENSE.md file distributed with the sources of this
+ * project regarding your rights to use or distribute this software.
```

Comment:
We can't remove the `Regents of the University of California...` section of the copyright. This file is coming from the previous version which was developed with that copyright header so it must remain.